### DISCUSSION TITLE
小错误

### DISCUSSION THREAD

**Discussion by yaohao159123** [AmazingAng/WTF-Solidity#485] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/74045902/233769223-83b40298-c9c6-4419-a7ce-f0f5f77d782b.png)
solidity301的34.ERC721有个小错误，将ERC165打成了ERC721

**Comment by Azleal** [comment, marked as answer]
原教程没错，这里说的就是`ERC721`实现`IERC165.suppurtsInterface()`
